modules/missenaBidAdapter.js

```diff
@@ -91,8 +92,10 @@ function toPayload(bidRequest, bidderRequest) {
   const bidFloor = getFloor(bidRequest);
   payload.floor = bidFloor?.floor;
   payload.floor_currency = bidFloor?.currency;
-  payload.currency = config.getConfig('currency.adServerCurrency') || 'EUR';
+  payload.currency = config.getConfig('currency.adServerCurrency');
   payload.schain = bidRequest.schain;
+  payload.coppa = config.getConfig('coppa') === true ? 1 : 0;
```

Comment:
can you please get coppa from `bidderRequest.ortb2.regs.coppa` instead?

Reply:
I have updated the code to get the `coppa` from the `bidderRequest`.

Reply:
@ChrisHuie is there anything else that we need to do here?

Reply:
looks good to me